config/services.yaml

```diff
@@ -1,4 +1,5 @@
 parameters:
+    env(ASSETS_ORIGIN_WHITELIST): '~.+~'
```

Comment:
One last thing @giorgiosironi, tempted to remove this as a default since it'll end up downloading _anything_ with an `xlink:href`, which would include references to DOIs, external links etc.

Reply:
I don't like this approach, but hard to determine what should be an asset (see https://jats.nlm.nih.gov/archiving/tag-library/1.2/attribute/xmlns-xlink.html). For example, `self-uri` may or may not be on ([JATS example](https://jats.nlm.nih.gov/archiving/tag-library/1.2/element/self-uri.html) isn't, but eLife uses it as the reference to the PDF, which is).

Reply:
I guess the alternative would be to configure a whitelist of tag names (such as `self-uri`), which is less useful in practice. So would be ok to make the the environment variable mandatory while removing the `.+` default?

The fact that it is a regex makes it flexible to use for any sort of prefix and more (e.g. different buckets on `https://s3.amazonaws.com/`, certain file extensions, etc)